evap/staff/views.py

```diff
@@ -914,6 +914,35 @@ def user_delete(request, user_id):
 
 
 @staff_required
+def user_merge_selection(request):
+    form = UserMergeSelectionForm(request.POST or None, request.FILES or None)
+
+    if form.is_valid():
+        main_user = form.cleaned_data['main_user']
+        other_user = form.cleaned_data['other_user']
+        return redirect('staff:user_merge', main_user.id, other_user.id)
+    else:
+        return render(request, "staff_user_merge_selection.html", dict(form=form))
```

Comment:
wouldn't it be helpful to have an additional button in the user list or in a userprofile's edit page "merge..." or "merge into..."? you have these pages open anyways when searching for the duplicates. but you are the user, so it's your call i guess dr